README.md

```diff
@@ -1,92 +1,74 @@
 
-##### Good Example
+- **Length**: Aim for 1-2 sentences (20-40 words).
+- **Clarity**: Highlight unique aspects of your pool, such as reliability, transparency or low fees.
+- **Strengths**: Mention features like high uptime, low fees, strong community support or eco-friendly practices.
+- **Tone**: Keep a friendly and professional tone.
 
-> "Every 12 hours"
-
-##### Bad Example
-
-> "Every 2 epochs"
-
-#### Logo
-
-- **Shape**: It is preferred that the logo follows the Nimiq hexagon shape. For reference, please refer to the bottom of the 'Colours' section of our [Nimiq Style Guide on Figma](<https://www.figma.com/design/GU6cdS85S2v13QcdzW9v8Tav/NIMIQ-Style-Guide-(Oct-18)?node-id=0-1&node-type=canvas&t=mNoervj6Kgw0KhKL-0>).
-- **Background**: Do not add a background colour to the image.
-- **Format**: Prefer to use SVG. File size isn't an issue as we will optimise and serve the best performing version (e.g. PNG, JPG, SVG) for each validator.
+**Good Example**: "Our pool offers 99.9% uptime, low fees, and strong security, ensuring a seamless staking experience. Join us for a reliable, transparent, and staker-focused service."
+**Bad Example**: "Stake with us for unmatched rewards and the highest profits! Don’t miss your chance to earn big — this is the ultimate opportunity for stakers!"
 
 ### Validator JSON schema
 
-You can use the [example JSON](./public/validators/main-albatross/.example.json) as a template.
+Use the following schema to create your validator information file. You can start by copying this [example JSON template](./public/validators/main-albatross/.example.json). When you add your validator information, you'll need to include specific keys in your JSON file. Below is an explanation of each key and its possible values.
 
-When you add your validator information, you'll need to include certain keys in your JSON file. Below is an explanation of each key and its possible values:
+**Required Keys**:
 
-- `address` (required): The address of the validator in the format `NQXX XXXX XXXX XXXX XXXX XXXX XXXX XXXX XXXX`.
-- `fee` (required): A number between 0 and 1 representing the fee percentage charged by your validator. For example, 0.05 represents a 5% fee.
-- `payoutType` (required): The method used to pay out validator rewards. Possible values are:
+- `name`: The name of your validator. If you don’t provide a name, your validator address will be used by default.
+- `description`: A short description of your validator. Use this to highlight what makes your validator unique or appealing to stakers.
+- `address`: The address of the validator in this format `NQXX XXXX XXXX XXXX XXXX XXXX XXXX XXXX XXXX`.
+- `fee`: A percentage fee between 0 and 1 (0.05 represents a 5% fee).
```

Comment:
Not sure if this is clear enough 🙈 .

Maybe overthinking, but something like this might be clearer?

```md
- Fee information. You need to add ONE of the following:
   - For static fees, use `fee`. A percentage fee between 0 and 1 (0.05 represents a 5% fee). 
   - For dynamic fees, you need to add three keys:
     - `feeLowest`: The minimum possible fee.
     - `feeHighest`: The maximum possible fee.
     - `feeDescription`: Outline the conditions for various fees.
```